style.css

```diff
@@ -106,28 +128,89 @@ blockquote {
   margin-left: 40px;
 }
 
-.vast-heading {
-  color: grey;
+h1, .vast-h1 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 2em;
+}
+
+h2, .vast-h2 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 1.5em;
+}
+
+h3, .vast-h3 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 1.17em;
+}
+
+h4, .vast-h4 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 1em;
+}
+
+h5, .vast-h5 {
+  color: #2f3733;
   font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: .83em;
+}
+
+h6, .vast-h6 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: .67em;
 }
 
```

Comment:
you can condense this further by putting all the similar attributes together and chaining the classes like so

```
h1,
h2,
h3,
h4,{
  color: #2f3733;
  font-family: 'Open Sans Condensed', sans-serif;
  etc
} ```

or even combine the similar parts into a "header" class